internal/acctest/config/extendedproperties/extended_properties_test.go

```diff
@@ -103,22 +103,26 @@ func testAccCheckExpectedExtendedPropertiesAttributes(extendedProperty *client.E
 			return err
 		}
 
+		// Check for no items if empty object sent
+		if extendedProperty.Name == nil {
+			if len(response.GetItems()) > 0 {
+				return fmt.Errorf("Empty items object sent, expected no items to be returned, but got %d", len(response.GetItems()))
+			}
+			return nil
+		}
+
 		// Verify that attributes have expected values
```

Comment:
This probably won't matter, but this could potentially miss an error. If we expect more than one item returned, this won't error out if none are returned. Because the checks only happen inside the if statement.